link_resolver.go

```diff
@@ -144,11 +144,15 @@ func doRequest(urlString string, r *http.Request) (interface{}, error, time.Dura
 	}
 
 	if isSupportedThumbnail(resp.Header.Get("content-type")) {
-		scheme := "https://"
-		if r.TLS == nil {
-			scheme = "http://" // https://github.com/golang/go/issues/28940#issuecomment-441749380
+		if *baseURL == "" {
+			scheme := "https://"
+			if r.TLS == nil {
+				scheme = "http://" // https://github.com/golang/go/issues/28940#issuecomment-441749380
+			}
+			response.Thumbnail = fmt.Sprintf("%s%s/thumbnail/%s", scheme, r.Host, url.QueryEscape(resp.Request.URL.String()))
```

Comment:
this is missing the prefix

Reply:
Prefix will always be an empty string here since it's based off baseURL (if it's set)